views/includes/page-templates/trust-legal-entity-bo.html

```diff
@@ -139,7 +139,20 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ pageParams.title }}</h1>
     {% set public_register_jurisdiction = formData.public_register_jurisdiction %}
     {% set registration_number = formData.registration_number %}
     {% include "includes/inputs/fields/is-on-register-in-country-formed-input.html" %}
-
+    
+    {% if formData.relevant_period %}
+    {% set fieldParam = {
+      name: 'startDate',
+      value: {
+        'Day': formData.startDateDay,
+        'Month': formData.startDateMonth,
+        'Year': formData.startDateYear
+      },
+      label: "When did they become a beneficiary?",
+      error: errors.startDate if errors
+    } %}
+    {% include "includes/inputs/date-input.html" %}
```

Comment:
maybe we can use `includes/inputs/date/start-date-input.html` instead? check out what they have done for start date for beneficial owners